august-first.json

```diff
@@ -1,12 +1,12 @@
 {
   "id": "august-first",
   "name": "August First",
-  "address": "115 St. Paul St. Burlington, VT 05401",
+  "address": "149 S Champlain St, Burlington, VT 05401",
```

Comment:
I see you found our _totally intentional, and in no way a mistake on our part_ messed up addresses.